packages/appstore/src/components/ready-to-update-wallets/upgrade-info-list.tsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import { Text, Icon } from '@deriv/components';
+import { Localize } from '@deriv/translations';
+import { isMobile } from '@deriv/shared';
+import { TReadyToUpgradeForm } from './ready-to-upgrade-form';
+
+type TUpgradeInformationList = {
+    name: string;
+    visiblity: boolean;
+    content: React.ReactElement;
+};
+
+const UpgradeInformationList = ({ is_eu }: TReadyToUpgradeForm) => {
```

Comment:
Can you make this to a config file

Reply:
I don't think I can because I am returning some components inside the file.